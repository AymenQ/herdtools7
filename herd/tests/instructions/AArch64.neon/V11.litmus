AArch64 V11
(* Tests load / store immediate unscaled, no offset *)
{ 
x=1;
0:X0=x; 0:X1=y
}

 P0              ;
 LDUR B0, [X0]   ;
 STUR B0, [X1]   ;
 forall (y=1)
 