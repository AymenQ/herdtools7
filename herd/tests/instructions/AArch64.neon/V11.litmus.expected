Test V11 Required
States 1
y=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (y=1)
Observation V11 Always 1 0
Hash=a2118731204b9841b881992e8d6f859d

